packages/allure-cucumberjs/src/BufferEncodingUtility.ts

```diff
@@ -0,0 +1,8 @@
+export const getBufferEncoding = (encoding: any): BufferEncoding => {
+  switch(encoding) {
+    case "BASE64":
```

Comment:
Let's use regexp, or turn the argument to lower case to make the function much more generic

Reply:
This function was unnecessary as Buffer.isEncoding() already validates a string